tests/functional_tests/test_groups/models/ernie_vl/ernie45_vl_vit_debug.py

```
@@ -0,0 +1,356 @@
+        # HF: norm1 then QKV
+        hf_normed = hf_vit.blocks[0].norm1(hf_patch_out)
+        hf_qkv = hf_vit.blocks[0].attn.qkv(hf_normed)
+
+        # MG: TELayerNormColumnParallelLinear does fused LN + Linear
+        # But we can compare the QKV output by accessing the linear_qkv layer
+        mg_linear_qkv = mg_layer0.self_attention.linear_qkv
+        # This is TELayerNormColumnParallelLinear - it fuses LN + Linear
+        mg_qkv_out, _ = mg_linear_qkv(mg_patch_out[:, None])  # [N, 1, 3*hidden]
+        mg_qkv_flat = mg_qkv_out.squeeze(1)
+
+    cos_sim_ln = F.cosine_similarity(
```

Flake8 (7.3.0)

[error] local variable 'cos_sim_ln' is assigned to but never used

(F841)